docs/content/concepts/terminology.md

```diff
@@ -1,100 +1,197 @@
+workspace may restrict the types of its children and may restrict the types it may be a child of; a parent-child
+relationship is allowed if and only if the parent allows the child and the child allows the parent.
+
+More information, including examples, can be found in the the [Workspaces document](../workspaces).
 
-A workspace binds APIs and makes them accessible inside the logical cluster, allocates capacity for creating instances
-of those APIs (quota), and defines how multi-workspace operations can be performed by users, clients, and controller
-integrations.
+### Workspace Types
 
-To a user, a workspace appears to be a Kubernetes cluster minus all the container orchestration specific resources. It
-has its own discovery, its own OpenAPI spec, and follows the kube-like constraints about uniqueness of
-Group-Version-Resource and its behaviour (no two GVRs with different schemas can exist per workspace, but workspaces can
-have different schemas). A user can define a workspace as a context in a kubeconfig file and `kubectl get all -A` would
-return all objects in all namespaces of that workspace.
+Workspaces have types, which are mostly oriented around a set of default or optional APIs exposed. For example:
 
-Workspace naming is chosen to be aligned with the Kubernetes Namespace object - a Namespace subdivides a workspace by
-name, a workspace subdivides the universe into chunks of meaningful work.
+- a workspace intended for deploying applications might expose the same API objects a user would encounter on a
+  physical cluster,
+- a workspace intended for building Knative functions might expose only the Knative serving APIs, ConfigMaps, Secrets,
+  and optionally enable Knative Eventing APIs.
 
-Workspaces are the containers for all API objects, so users orient by viewing lists of workspaces from APIs.
+By default, each workspace has [built-in APIs installed and available to its users](./apis/built-in).
 
-## Workspace Type
+More information, including a list of Workspace Types and examples, can be found in the
+[Workspace Types document](../workspaces/workspace-types/).
 
-Workspaces have types, which are mostly oriented around a set of default or optional APIs exposed.  For instance, a
-workspace intended for use deploying Kube applications might expose the same API objects a user would encounter on a
-physical cluster.  A workspace intended for building functions might expose only the knative serving APIs, config maps
-and secrets, and optionally enable knative eventing APIs.
+### Virtual Workspace
 
-At the current time there is no decision on whether a workspace type represents an inheritance or composition model,
-although in general we prefer composition approaches.  We also do not have a fully resolved design.
+Virtual Workspaces are proxy-like API servers under a custom URL that provide a computed view of real Workspaces.
```

Comment:
I think this sentence (from further down, which is currently being deleted) should be added here:

```suggestion
Virtual Workspaces are proxy-like API servers under a custom URL that provide a computed view of real Workspaces.
Those workspaces are “virtual” because they adapt or transform the underlying source of truth for the
object and potentially the schema the user sees.
```